core/network/src/behaviour.rs

```diff
@@ -109,6 +117,11 @@ impl<B: BlockT, S: NetworkSpecialization<B>, H: ExHashT> Behaviour<B, S, H> {
 	pub fn put_value(&mut self, key: record::Key, value: Vec<u8>) {
 		self.discovery.put_value(key, value);
 	}
+
+	/// Get unique access to the light client handler.
+	pub fn light_client_handler(&mut self) -> &mut protocol::LightClientHandler<Substream<StreamMuxerBox>, B> {
```

Comment:
I don't think this is used anywhere, so I'd prefer to remove it, otherwise someone is going to use this method. The `LightClientHandler` is supposed to be an implementation detail, so not accessible directly.

Reply:
How do you want to expose the request functionality of the light client handler? Do you want to reproduce its `request` method here and forward the call?

Reply:
> Do you want to reproduce its request method here and forward the call?

Yes!